cmd/krew/cmd/version.go

```diff
@@ -18,6 +18,7 @@ import (
 	"fmt"
 	"os"
 
+	"github.com/GoogleContainerTools/krew/pkg/version"
```

Comment:
please split in 3 regions,
```
import(
stdlib...

project...

vendor...
)
```

Reply:
I easily found more examples of this already in the repo (se we’re already not there) and as far as I know this is not a convention  + no linter check has failed.